MacrobenchmarkSample/macrobenchmark/src/main/AndroidManifest.xml

```diff
@@ -15,10 +15,21 @@
   -->
 
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
     package="com.example.macrobenchmark">
 
+    <!--
+        Legacy external storage + storage permission used so CI can write outputs to e.g. /sdcard/.
+        This should ideally only required on the androidTest, but Samsung devices seem to expect
+        this.
+    -->
+    <application android:requestLegacyExternalStorage="true" />
+
     <queries>
         <package android:name="com.example.macrobenchmark.target" />
     </queries>
-    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
+
+    <uses-permission
```

Comment:
Library defines this (https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:benchmark/common/src/main/AndroidManifest.xml?q=android.permission.WRITE_EXTERNAL_STORAGE&ss=androidx%2Fplatform%2Fframeworks%2Fsupport&start=21)

Why is it needed here? Does the ignore just need to be here?